superset/common/query_object.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         extras: Optional[Dict[str, Any]] = None,
         columns: Optional[List[str]] = None,
         orderby: Optional[List[List[str]]] = None,
-        post_processing: Optional[List[Dict[str, Any]]] = None,
+        post_processing: Optional[List[Optional[Dict[str, Any]]]] = None,
```

Comment:
This is starting to deserve a type definition of it's own

Reply:
Actually this is an intermediate structure; the `Optional`s are removed in the constructor, resulting in `List[Dict[str, Any]]]`. But you're right, this is becoming difficult to follow, will think about a good solution to adding types here.